src/stats.rs

```diff
@@ -403,26 +499,37 @@ impl Collector {
             // Some are counters, some are gauges...
             match stat.name {
                 EventName::Query => {
+                    self.statsd_client.send_increment("query_count");
+
                     let counter = stats.entry("total_query_count").or_insert(0);
                     *counter += stat.value;
                 }
 
                 EventName::Transaction => {
+                    self.statsd_client.send_increment("tx_count");
+
                     let counter = stats.entry("total_xact_count").or_insert(0);
                     *counter += stat.value;
                 }
 
                 EventName::DataSent => {
+                    self.statsd_client.send_count("bytes_sent", stat.value);
```

Comment:
You want to use "standard pgbouncer" names for these, not new ones that most monitoring tools wouldn't expect.

Reply:
pgbouncer metric [here](https://docs.sysdig.com/en/docs/sysdig-monitor/metrics-dictionary/metrics-in-sysdig-legacy-format/applications/pgbouncer-metrics/) is `bytes_sent_per_second`. Since we want statsd to do all the aggregation we don't need to add the per second piece.

Reply:
I'm just saying that you'll be sending "non-standard" stats via statsd and that they will also differ from what's in the admin database. Is this desired?

Reply:
@drdrsh thoughts?

Reply:
My take on the metric submissions through statsd is that we'll want to send raw metric data and leave it up to the receiver to decide how to aggregate/sample/transform. The admin DB produces a view of these metrics and pgbouncer had to aggregate on its own to make the data useful but with external statsd servers, we don't have to pre-aggregate